Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionView.swift

```diff
@@ -151,7 +169,8 @@ public struct FaceLivenessDetectorView: View {
                     guard displayState != .displayingLiveness else { return }
                     displayState = .displayingLiveness
                 },
-                beginCheckButtonDisabled: false
+                beginCheckButtonDisabled: false,
+                challenge: viewModel.challenge!
```

Comment:
is it possible to avoid force unwrapping an optional? this can lead to NPE crashes